src/Middleware/WorkflowFiltersProviderInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace ObjectivePHP\Application\Middleware;
+
+/**
+ * Interface MiddlewareFiltersProviderInterface
+ *
+ * @package ObjectivePHP\Application\Middleware
+ */
+interface WorkflowFiltersProviderInterface extends MiddlewareInterface
+{
+    /**
+     * Run the embedded filters
+     *
+     * @return bool
+     */
+    public function runFilters(): bool;
```

Comment:
WorkflowFiltersProvider should exposer something like a getFilters() abstract method, not runFilters... the idea is that a Middleware can be bundled together with some default filters, to be merged with those set at run time, when plugging the middleware into the application.

By the way, such an interface also should expose clearFilters() to let the developer decide whether they wants or not to make use of the default filters.